examples/routing_service/routing_service_with_security/README.md

```diff
@@ -0,0 +1,75 @@
+# Routing Service with Security example
+
+## Concept
+Routing Service supports DDS security, and may be configured to bridge 
+topic data between:
+- 2 unsecured domains (no DDS security) or 
+- 1 secured domain and 1 unsecured domain or
+- 2 secured domains
+
+## Example description
+This example demonstrates Routing Service bridging 
+1 secured domain and 1 unsecured domain using the RTI DDS Ping utility.
+The rtiddsping publisher resides in Domain 0, an unsecured domain and 
+the rtiddsping subscriber resides in Domain 1, a secured domain.
+
+---------------------------          ---------------------------
```

Comment:
You may want to put this chart inside triple grave accents (```), so everything is fixed width.

Reply:
Done.